runtime/doc/diagnostic.txt

```diff
@@ -462,37 +462,39 @@ get_next({opts})                                   *vim.diagnostic.get_next()*
     Get the next diagnostic closest to the cursor position.
 
     Parameters: ~
-      • {opts}  (table) See |vim.diagnostic.goto_next()|
+      • {opts}  nil|table See |vim.diagnostic.goto_next()|
```

Comment:
why `nil|table` instead of the more common `table|nil`? 

Also, the parens are deliberate: lua-language-server ignores them, while gen_vimdoc uses them for better-looking docs.

Reply:
> why nil|table instead of the more common table|nil?

No real reason,  I'll change it

> Also, the parens are deliberate

I didn't remove any from the source,  it happened in the doc generation?

Reply:
> I didn't remove any from the source, it happened in the doc generation?

Hmm, possibly the `nil|table` confused it if the script is too clever. They might come back (or stay then for `(table|nil)` in the source.

Reply:
> They might come back 

Looks like they did come back for `table|nil`, but not for `Diagnostic|nil`

Reply:
looks like the script is case sensitive?